org.osgi.service.feature/bnd.bnd

```diff
@@ -4,4 +4,5 @@
 Export-Package: ${p}.*; -split-package:=first
 
 -buildpath: \
-    osgi.annotation;maven-scope=provided;version=${osgi.annotation.version}
+	osgi.annotation;maven-scope=provided;version='${osgi.annotation.version}',\
+	org.osgi.namespace.implementation
```

Comment:
You must specify a version 1.0 on here like is done in other bnd files.

Reply:
fixed